CHANGELOG.md

```diff
@@ -10,6 +10,9 @@ Every entry has a category for which we use the following visual abbreviations:
 
 ## [Unreleased]
 
+- 🎁 VAST gained support for importing suricata json logs. Specify the
+  `suricata` format to the `import` command and you're in business.
```

Comment:
I like being in business! 😄 I still think we should write the changelog in a very clear and sober style. Here's an alternate suggestion.

> VAST gained support for importing Suricata JSON logs. The `import` command has a new `suricata` format that can ingest EVE JSON output.